swift/src/Ice/InputStream.swift

```diff
@@ -764,16 +754,15 @@ extension InputStream {
 
     /// Reads a proxy from the stream (internal helper).
     ///
-    /// - returns: `ProxyImpl?` - The proxy read from the stream.
+    /// - Returns: The proxy read from the stream.
     public func read<ProxyImpl>() throws -> ProxyImpl? where ProxyImpl: ObjectPrxI {
         return try ProxyImpl.ice_read(from: self)
     }
 
     /// Reads an optional proxy from the stream (internal helper).
     ///
-    /// - parameter tag: `Int32` - The tag of the optional data member or parameter.
-    ///
-    /// - returns: `ProxyImpl?` - The proxy read from the stream.
+    /// - Parameter tag: The tag of the optional data member or parameter.
```

Comment:
We should replace "data member" by field when talking about Slice.

Reply:
Will do in follow-up PR.